### PR TITLE
WIP:proposal review builder

### PR DESCRIPTION
**Pull request by sbillinge**
the manuscreipt review builder is a crutch to ease the refereeing of manuscripts.  Now I want something similar for reviewing proposals.....

No need to look at this yet, there is still a way to go.

### CODE REVIEW

**Comment by scopatz** [comment]
LGTM CC @CJ-Wright

File: regolith/builders/proposalreviewbuilder.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+"""Builder for Current and Pending Reports."""
```
Comment (CJ-Wright) [inline regolith/builders/proposalreviewbuilder.py]:
```suggestion
"""Builder for Proposal Reivews."""
```

File: regolith/builders/proposalreviewbuilder.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import datetime
+import time
+from nameparser import HumanName
```
Comment (CJ-Wright) [inline regolith/builders/proposalreviewbuilder.py]: I think this needs to be added to the requirements

File: regolith/builders/proposalreviewbuilder.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+from regolith.sorters import position_key
+from regolith.tools import (
+    all_docs_from_collection,
+    filter_grants,
+    fuzzy_retrieval,
+)
+
+
+class PropRevBuilder(LatexBuilderBase):
+    """Build a manuscript review from database entries"""
```
Comment (CJ-Wright) [inline regolith/builders/proposalreviewbuilder.py]:
```suggestion
    """Build a proposal review from database entries"""
```

File: regolith/builders/proposalreviewbuilder.py (hunk 2 above)
Comment (sbillinge) [reply regolith/builders/proposalreviewbuilder.py]: OK, I added to ``run.txt`` in ``requirements``.  Let me know if it should be removed.

**Comment by scopatz** [comment]
let us know when this is no longer a WIP

**Comment by sbillinge** [comment]
this builds well locally.  I need to work on the schema.py before it is ready for merge.

**Comment by sbillinge** [comment]
force push was to change a commit message, but I also inadvertently had a merge of upstream master in there so I hope I didn't break anything.....a little worried about that.....

**Comment by scopatz** [comment]
Seems fine!

**Pull request merged**